src/main.py

```diff
@@ -31,6 +34,11 @@
     # Source name: Binary encoded
     train_binary_encoded_source = get_binary_encoded_feature(train_source_name_feature)
 
+    # get stemmed/sorted job title and job title modifiers
+    train_job_titles, train_job_title_modifiers = get_stemmed_sentences(
+        pre_process_job_titles()
```

Comment:
Related to the other comment: `pre_process_job_titles(title_feature)`  passed from `train_raw_data`